source/linux/Overview/_Top_Level_Makefile.rst

```diff
@@ -201,7 +201,7 @@ assume that you are calling the Makefile from the top-level of the SDK.
    ::
 
       BOOT_PART?=/path/to/sd-card/boot-partition
-      ROOT_PART?=/path/to/sd-card/root-partition
+      ROOTFS_PART?=/path/to/sd-card/root-partition
```

Comment:
Other parts of this file still reference the `ROOT_PART` variable.

Reply:
Fixed this